app/livechat/client/views/app/livechatCustomFields.js

```diff
@@ -49,6 +56,7 @@ Template.livechatCustomFields.events({
 	},
 });
 
-Template.livechatCustomFields.onCreated(function() {
-	this.subscribe('livechat:customFields');
+Template.livechatCustomFields.onCreated(async function() {
```

Comment:
no need to be `async` here
```suggestion
Template.livechatCustomFields.onCreated(function() {
```